documentation/docs/07-hooks.md

```diff
@@ -79,47 +115,37 @@ export async function handle({ event, resolve }) {
 }
 ```
 
-### handleError
+#### handleError
 
-If an error is thrown during loading or rendering, this function will be called with the `error` and the `event` that caused it. This allows you to send data to an error tracking service, or to customise the formatting before printing the error to the console.
-
-During development, if an error occurs because of a syntax error in your Svelte code, a `frame` property will be appended highlighting the location of the error.
+If an error is thrown during loading or rendering, this function will be called with the `error` and the `event` that caused it. This allows for two things:
 
-If unimplemented, SvelteKit will log the error with default formatting.
+- you can send data to an error tracking service, for example
+- you can customise the formatting. The return value of the function becomes the value of `$page.error`. To make this type-safe, you can customize the expected shape through `App.PageError`. Note that `message` is always a required property, to guarantee a sensible fallback behavior.
 
 ```js
-/// file: src/hooks.js
+/// file: src/hooks.server.js
 // @filename: ambient.d.ts
 const Sentry: any;
 
 // @filename: index.js
 // ---cut---
-/** @type {import('@sveltejs/kit').HandleError} */
+/** @type {import('@sveltejs/kit').HandleServerError} */
 export function handleError({ error, event }) {
 	// example integration with https://sentry.io/
 	Sentry.captureException(error, { event });
+	return { message: /** @type {Error} */(error).message }
 }
 ```
 
-> `handleError` is only called for _unexpected_ errors. It is not called for errors created with the [`error`](/docs/modules#sveltejs-kit-error) function imported from `@sveltejs/kit`, as these are _expected_ errors.
-
-### handleFetch
+If unimplemented or no error shape is returned, SvelteKit use the default formatting `{ message: string }`.
 
-This function allows you to modify (or replace) a `fetch` request that happens inside a `load` function that runs on the server (or during pre-rendering).
+During development, if an error occurs because of a syntax error in your Svelte code, a `frame` property will be appended highlighting the location of the error.
```

Comment:
Is this still true? Feels like this stuff should be kept out of the error object

```suggestion
```

Reply:
Mhm not sure, but I feel like it should (hopefully?) still be true - only that it's passed in, but not necessarily appears in your user-facing error page.

Reply:
Ah right, I mistook input for output. Sleep deprivation